manifests/crc/default-db-included-installation/openshift/kruize-crc-openshift.yaml

```diff
@@ -119,9 +119,9 @@ data:
         {
           "name": "prometheus-1",
           "provider": "prometheus",
-          "serviceName": "",
-          "namespace": "",
-          "url": "https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091"
+          "serviceName": "prometheus-k8s",
```

Comment:
@shreyabiradar07 Include tests to validate this

Reply:
PR [#1291](https://github.com/kruize/autotune/pull/1291) adds support for `servicename` and `namespace` as input parameters for validating datasource manifest configuration.